tests/e2e-kubernetes/testsuites/credentials.go

```diff
@@ -50,12 +56,22 @@ const (
 	stsAssumeRoleRetryMaxBackoffDelay = 10 * time.Second
 )
 
+const (
+	eksauthAssumeRoleRetryCode            = "AccessDeniedException"
+	eksauthAssumeRoleRetryMaxAttemps      = 0 // This will cause SDK to retry indefinetly, but we do have a timeout on the operation
+	eksauthAssumeRoleRetryMaxBackoffDelay = 10 * time.Second
+)
+
 const serviceAccountTokenAudienceSTS = "sts.amazonaws.com"
 const roleARNAnnotation = "eks.amazonaws.com/role-arn"
 const credentialSecretName = "aws-secret"
 
+const serviceAccountTokenAudienceEKS = "pods.eks.amazonaws.com"
+
 // DefaultRegion specifies the STS region explicitly.
 var DefaultRegion string
+var ClusterName string
+var IsPodMounter bool
```

Comment:
We already have this variable defined in `pod_sharing.go`, I think we just need to remove this definition